templates/zerver/help/add-or-remove-users-from-a-stream.md

```diff
@@ -17,13 +17,20 @@ to a stream][configure-invites].
 
 1. Select a stream.
 
+1. Select the **Subscribers** tab on the right.
+
 1. Under **Add subscribers**, enter a name or email address. The typeahead
    will only include users who aren't already subscribed to the stream.
 
 1. Click **Add**.
 
 {end_tabs}
 
+!!! tip ""
+
+      If you want to bulk add users, you can copy the membership from
```

Comment:
Maybe:

"To add users in bulk, you can copy members from an existing stream or [user group](/help/user-groups)."